src/Contrib/Jaeger/SpanConverter.php

```diff
@@ -29,10 +32,34 @@ public function convert(SpanDataInterface $span): JTSpan
         $references = $tags = $logs = [];
         $startTime = AbstractClock::nanosToMicro($span->getStartEpochNanos());
         $duration = AbstractClock::nanosToMicro($span->getEndEpochNanos() - $span->getStartEpochNanos());
-        $tags = [
-            self::STATUS_CODE_TAG_KEY => $span->getStatus()->getCode(),
-            self::STATUS_DESCRIPTION_TAG_KEY => $span->getStatus()->getDescription(),
-        ];
+
+        $tags = [];
+
+        if ($span->getStatus()->getCode() !== StatusCode::STATUS_UNSET) {
+            switch ($span->getStatus()->getCode()) {
+                case StatusCode::STATUS_OK:
+                    $tags[self::STATUS_CODE_TAG_KEY] = 'OK';
```

Comment:
these should be constants... eg STATUS_CODE_OK and STATUS_CODE_ERROR

Reply:
This is a rather strange one. Java and JS are both using enums that are using full caps and so are able to refer directly back to the name for this, whereas Go is using normal case like PHP is and so has to do this too.

I tried to allude to that in the naming of the constants, but if more detailed comments around that are needed lmk

Reply:
*I'm unable to tell from the specs which is correct, as this one I followed for this PR suggests that it's the all caps version, but then the one linked to in the PHP class's comment seems to say go with the normal case version

Reply:
FWIW the spec suggesting all caps appears to be around 6 months more recently edited than the the one linked in the StatusCode class file, so maybe there was some changes around this that weren't fully updated? Just a guess really

Reply:
a real-world test will get to the bottom of it, and we're still pre-alpha so there's plenty of opportunity for people to test it out. I suspect that the constants in `StatusCode` are correct for otel, but doesn't mean that Jaeger would use those internally, so let's stick with uppercase.